packages/webex/src/calling.js

```diff
@@ -1,12 +1,10 @@
-import {
-  createClient,
-  createCallHistoryClient,
-  createVoicemailClient,
-  createContactsClient,
-  createCallSettingsClient,
-} from '@webex/calling';
+import CallingSdk from '@webex/calling';
```

Comment:
Would WebexCalling be a more appropriate name? The calling sdk is not really a separate sdk any more. it's a package within js-sdk.

Reply:
Also, why is this file located inside webex/src/ ? Can we not keep it under packages/calling?

Reply:
all the global CDN reference are on webex , we already have meetings being exported on the webex name space

Reply:
Importing webex object inside packages/calling was not working as we were getting error from crypto module in samples page the moment we import webex. Hence for webex initialization to be done inside calling, we needed it to move it under webex/src